config.json

```diff
@@ -1258,6 +1258,17 @@
         "object_oriented_programming"
       ]
     },
+    {
+      "uuid": "adad6be5-855d-4d61-b14a-22e468ba5b44",
+      "slug": "hangman",
+      "core": false,
+      "unlocked_by": null,
+      "difficulty": 5,
+      "topics":[
+        "events",
+        "reactive_programming"
```

Comment:
I would suggest adding `strings` to this list, as there is inherently some string manipulation involved with this exercise.